functions.php

```diff
@@ -12,94 +12,89 @@
  * @package air-light
  */
 
+namespace Air_Light;
+
 /**
  * The current version of the theme.
  */
 define( 'AIR_LIGHT_VERSION', '4.9.9' );
 
 /**
- * Requires.
+ * Theme settings
  */
-require get_theme_file_path( '/inc/functions.php' );
-require get_theme_file_path( '/inc/menus.php' );
-require get_theme_file_path( '/inc/nav-walker.php' );
 
-/**
- * Enable theme support for essential features.
- */
-add_theme_support( 'automatic-feed-links' );
-add_theme_support( 'title-tag' );
-add_theme_support( 'post-thumbnails' );
-add_theme_support( 'html5', array( 'search-form', 'comment-form', 'comment-list', 'gallery', 'caption' ) );
-// add_theme_support( 'woocommerce' );
+$theme_settings = [
+  // Set image sizes & content width
+  'image_sizes' => [
+    'small' => 300,
+    'medium' => 700,
+    'large' => 1200,
+  ],
+  'content_width' => 800,
 
-/**
- * Load textdomain.
- */
-load_theme_textdomain( 'air-light', get_template_directory() . '/languages' );
+  'default_featured_image' => get_theme_file_uri( 'images/default.jpg' ),
 
-/**
- * Define content width in articles
- */
-if ( ! isset( $content_width ) ) {
-  $content_width = 800;
-}
+  'logo_path' => get_theme_file_path( '/svg/logo.svg' ),
+  'logo_url' => get_theme_file_uri( '/svg/logo.svg' ),
 
-/**
- * Register widget area.
- *
- * @link https://developer.wordpress.org/themes/functionality/sidebars/#registering-a-sidebar
- */
-add_action( 'widgets_init', 'air_light_widgets_init' );
-function air_light_widgets_init() {
-  register_sidebar( array(
-    'name'          => esc_html__( 'Sidebar', 'air-light' ),
-    'id'            => 'sidebar-1',
-    'description'   => esc_html__( 'Add widgets here.', 'air-light' ),
-    'before_widget' => '<section id="%1$s" class="widget %2$s">',
-    'after_widget'  => '</section>',
-    'before_title'  => '<h2 class="widget-title">',
-    'after_title'   => '</h2>',
-  ) );
-} // end air_light_widgets_init
+  // Set theme support
+  'theme_support' => [
+    'automatic-feed-links' => [],
+    'title-tag' => [],
+    'post-thumbnails' => [],
+    // 'woocommerce' => [],
```

Comment:
In my opinion, we can drop every WC related thing from Air light completely as those are needed so rarely.